docs/MeshGeneration/StructuredMesh.md

```diff
@@ -0,0 +1,103 @@
+  type(Geometry1D),target :: geometry
+
+  call mesh % StructuredMesh(nElem=20, &
+                             x=(/0.0_prec,1.0_prec/))
+
+  call interp % Init(N=7, controlNodeType=GAUSS, &
+                     M=10, targetNodeType=UNIFORM)
+
+  call geometry % Init(interp,mesh%nElem)
+  call geometry % GenerateFromMesh(mesh)
+
+```
+
+Notice that initializing the geometry requires an interpolant and the number of elements as input. 
+
+!!! note
+    Under the hood, the interpolant for the geometry (`geometry % interp` ) is associated with a pointer to `interp`, ie `geometry % interp => interp`.
+
+Once the geometry is initialized, the physical positions and metric terms can be calculated and stored using the `GenerateFromMesh` method.
+
+## Two Dimensions (2-D)
+To generate a structured mesh in one dimension, use the `StructuredMesh` generic in the [`Mesh2D`](../ford/type/mesh2d.html) class. 
+
+At the moment, only uniformly space structured meshes of elements can be generated. This means that all of the elements are of the same width; keep in mind that within each element, there is a quadrature grid. The points in the quadrature grid are spaced so that spectral accuracy is guaranteed.
+
+SELF uses a tiled structured grid. Tiled grids divide the 2-D grid into `nTilex`x`nTiley` tiles of size `nxPerTile`x`nyPerTile` . The width and height of the elements are defined as `dx` and `dy`. With these parameters,
+
+* `nx = nTilex*nxPerTile` is the total number of elements in the x-direction
+* `ny = nTiley*nyPerTile` is the total number of elements in the y-direction
+* `nelem = nx*ny` is the total number of elements
+* `Lx = dx*nx` is the domain length in the x-direction
+* `Ly = dy*ny` is the domain length in the y-direction
+
+You can set boundary conditions for each of the four sides of the structured mesh using a 1-D array of integers of length 4. The boundary conditions must be provided in counter-clockwise order, starting with the "south" boundary (south, east, north, west). The following built-in flags are available for setting boundary conditions
+
+* `SELF_BC_NONORMALFLOW`
+* `SELF_BC_PRESCRIBED`
+* `SELF_BC_RADIATION`
+
+The tiled layout is convenient for domain decomposition, when you are wanting to scale up your application for distributed memory platforms. You can further enable domain decomposition by setting the optional `enableDomainDecompisition` input to `.true.` . In this case, when you launch your application with `mpirun`, the domain will be automatically divided as evenly as possible across all MPI ranks.
+
+!!! note
+    It's good practice to set the total number of tiles equal to the number of MPI ranks that you are running with. Alternatively, you can use fairly small tiles when working with a large number of MPI ranks to increase the chance of minimizing point-to-point communications .
+
+In the example below, we create a 2-D mesh with the following attributes
+
+* $2 × 2$ elements for the domain
```

Comment:
2 x 2 elements or tiles per domain?

Reply:
2x2 tiles for the domain.

Reply:
Resolved in [528a97e](https://github.com/FluidNumerics/SELF/pull/61/commits/528a97e2201e486a90121bcfebd12b756a191679)